src/main.py

```diff
@@ -82,30 +83,40 @@ def main(module_name: OracleModule):
     logger.info({'msg': 'Sanity checks.'})
     check_providers_chain_ids(web3)
 
-    if module_name == OracleModule.ACCOUNTING:
+    if module == OracleModule.ACCOUNTING:
         logger.info({'msg': 'Initialize Accounting module.'})
         accounting = Accounting(web3)
         accounting.check_contract_configs()
         accounting.run_as_daemon()
-    elif module_name == OracleModule.EJECTOR:
+    elif module == OracleModule.EJECTOR:
         logger.info({'msg': 'Initialize Ejector module.'})
         ejector = Ejector(web3)
         ejector.check_contract_configs()
         ejector.run_as_daemon()
+    elif module_name == OracleModule.READINESS:
```

Comment:
READINESS -> to test

Reply:
fixed

Reply:
<img width="1077" alt="image" src="https://user-images.githubusercontent.com/31664571/228442028-edc91142-d0a4-4fa9-8978-7dee24c12424.png">